game/test/contrib/components/InventoryComponentTest.java

```diff
@@ -138,4 +147,46 @@ public void getAllItemsInventoryNoAddedItemButCreated() {
         assertEquals("should have no Items", 0, list.size());
         assertFalse("Item should be in returned List", list.contains(itemData));
     }
+
+    @Test
+    public void tranfserItem() {
+        InventoryComponent ic = new InventoryComponent(new Entity(), 1);
+        InventoryComponent other = new InventoryComponent(new Entity(), 1);
+        ItemData item = Mockito.mock(ItemData.class);
+        ic.add(item);
+        assertTrue(ic.items().contains(item));
```

Comment:
Bin mal wieder kleinlich zum später besser lesen können des Fehlers gibt es häufiger.
Da die bool asserts keine gute Fehler nachricht haben. Sondern nur sagen AssertError.

```suggestion
        assertTrue("Componente enthält item nach hinzufügen",ic.items().contains(item));
```

Reply:
hab überall etwas text hinzugefügt